makefiles/Makefile.docs

```diff
@@ -86,6 +86,22 @@ next-gen-stage: ## Host online for review
 		echo "Hosted at ${STAGING_URL}/${MUT_PREFIX}/${USER}/${GIT_BRANCH}/"; \
 	fi
 
+next-gen-deploy:
```

Comment:
should we add this? cant we re use the one in shared.mk? i assume since we define CUSTOM_SEARCH_INDEX, next-gen-deploy-search-index defined here will be called?

Reply:
Reasonable question! I was worried the "local" next-gen-deploy-search-index wouldn't be called, but I'm not sure why. We can give it a shot without this dupe.

Reply:
Updated :)